services/QuillLMS/engines/evidence/app/models/evidence/prompt.rb

```diff
@@ -68,6 +68,27 @@ def conjunctions
       [conjunction]
     end
 
+    def first_passage = activity.passages.first
+
+    def plagiarism_text
+      plagiarism_rule&.plagiarism_texts&.first&.text || ""
+    end
+
+    private def plagiarism_rule
+      rules&.find_by(rule_type: Evidence::Rule::TYPE_PLAGIARISM)
+    end
+
+    def distinct_highlight_texts
```

Comment:
This method includes the `auto_ml` scope.  Should the method name reflect this so that someone who comes along later doesn't assume that this finds all highlights, even those in other rule categories such as regex?  (I don't know that we use highlights in other rule types, but the models and admin UI do allow it.)